src/components/adminArea/Drawer/UserEditRoles.tsx

```diff
@@ -24,6 +24,16 @@ const UserEditRoles = ({ currentUser, role, setUserRole, shouldEdit }) => {
             ? filterObjectByKeys(currentUser?.role, [nameSpace])
             : currentUser?.role;
 
+    const getBorderColor = (key) =>
```

Comment:
Suggestion (non-blocking): Both `getBorderColor` and `getRadioColor` functions have the same purpose and return the same value, consider creating a unique function named `getComponentColor` to return their colors